### PR TITLE
App status ui 1052553

### PR DESCRIPTION
**Pull request by elysium001**
Please review, thanks!

### CODE REVIEW

**Comment by elysium001** [comment]
screen shot 2015-09-14 at 9 58 55 pm

File: media/css/devreg/status.styl [hunk 1]
```diff
@@ -1,5 +1,28 @@
 @import 'lib';
 
+#disable-addon {
+    padding: 10px 7px;
+    background: $gray;
+    width: 100px;
```
Comment (atiqueahmedziad) [inline media/css/devreg/status.styl]: please arrange the css properties in alphabetic order

File: media/css/devreg/status.styl [hunk 2]
```diff
@@ -1,5 +1,28 @@
 @import 'lib';
 
+#disable-addon {
```
Comment (ngokevin) [inline media/css/devreg/status.styl]: classes are usually preferred over IDs for styling since IDs have too much specificity

**Comment by ngokevin** [comment]
couple of tests need updating:

FAIL: mkt/developers/tests/test_views_ownership.py:TestDeveloperRoleAccess.test_disable
FAIL: mkt/developers/tests/test_views_ownership.py:TestDeveloperRoleAccess.test_enable

since the IDs changed

**Pull request closed without merging**